styles/main.scss

```diff
@@ -604,10 +599,6 @@ header {
   border: 1px solid transparent;
 }
 
-.posts-list-block:hover .content:after {
-  border-color: #fe6c0a;
-}
-
```

Comment:
Why were those lines removed?

Reply:
That snippet has just been removed a little bit lower, for better readability, you mentioned it at comment above🤔

Reply:
@1alexvash

Reply:
Which snippet
?

Reply:
.posts-list-block:hover .content:after {
  border-color: #fe6c0a;
}

Reply:
You just moved this piece of code?